owncloudTestUtil/src/main/java/com/owncloud/android/testutil/livedata/LiveDataUtils.kt

```diff
@@ -33,8 +33,8 @@ inline fun <reified T> LiveData<T>.getEmittedValues(
     val data = arrayOfNulls<T>(expectedSize)
     val latch = CountDownLatch(expectedSize)
     val observer = object : Observer<T> {
-        override fun onChanged(o: T?) {
-            data[currentValue.getAndAdd(1)] = o
+        override fun onChanged(value: T) {
```

Comment:
Is it true that `T` cannot be null here? You removed the nullability

Reply:
I'm not sure why removed, I leave with the null